config/locales/exceptions.en.yml

```diff
@@ -139,10 +214,11 @@ en:
         <<: *validation_errors
         code: 'RX157'
         detail: 'Email address is invalid'
-      SM152:
-        <<: *validation_errors
-        code: 'SM152'
-        detail: 'Email address is invalid'
+      MHVAcctCreation155: # Account is already Upgraded
```

Comment:
Is this tested with a spec? In my experience adding APPEALSSTATUS404, the key should be in all uppercase because the i18n key resolution upcases the client name when looking up.